lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/producers/ChecklistProducer.java

```diff
@@ -114,15 +114,14 @@ public void fillComponents(UIContainer tofill, ViewParameters viewparams, Compon
                 UIInput.make(form, "csrf", "simplePageBean.csrfToken", sessionToken.toString());
 
             // Name of the checklist
-            UIInput.make(form, "name", "#{simplePageBean.name}").decorate(new UIFreeAttributeDecorator("style", "vertical-align:middle;"));
+            UIInput.make(form, "name", "#{simplePageBean.name}");
```

Comment:
Thanks for the feedback Leonardo, I didn't intend to remove the vertical-align so have added it to the stylesheet.